generated/nifake/nifake_service.cpp

```diff
@@ -46,38 +42,6 @@ namespace nifake_grpc {
     }
   }
 
-  void NiFakeService::Copy(const CustomStruct& input, nifake_grpc::FakeCustomStruct* output) 
```

Comment:
The scalar copy methods are now generated as `convert_to/from_grpc` methods in the same file but now in the converters namespace (at the end of the file) instead of as class members.

The Array copy methods are no longer generated. They were identical and are now in converters.h.